include/arch/arm64/structs.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) BayLibre SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_ARM64_STRUCTS_H_
+#define ZEPHYR_INCLUDE_ARM64_STRUCTS_H_
```

Comment:
FWIW: there is already a "kernel_arch_data.h" file that could probably do this job.  Not a reason not to merge, but our arch headers remain sort of a problem we should clean up.